.gitignore

```diff
@@ -2,3 +2,6 @@
 .idea/
 build/
 vendor/
+/*.pem
+.dcos_superuser_password_hash
+/.vscode/
```

Comment:
You should put this in your `~/.gitignore` but i'll let it slide... this time.

Reply:
:laughing: what's the difference between this and `.idea/`?

Reply:
I wasn't here for the review 😎